Sources/HipMobileUI/Pages/CharacterSelectionPage.xaml

```diff
@@ -1,37 +1,58 @@
+// limitations under the License.
+-->
 
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:helpers="clr-namespace:PaderbornUniversity.SILab.Hip.Mobile.UI.Helpers;assembly=HipMobileUI"
+             xmlns:views="clr-namespace:PaderbornUniversity.SILab.Hip.Mobile.UI.Views;assembly=HipMobileUI"
              x:Class="PaderbornUniversity.SILab.Hip.Mobile.UI.Pages.CharacterSelectionPage">
     <ContentPage.Content>
-        <Grid x:Name="MainGrid" RowSpacing="0" ColumnSpacing="0">
-            <Grid x:Name="AdventurerGrid" BackgroundColor="Orange">
-                <Grid.GestureRecognizers>
-                    <TapGestureRecognizer Command="{Binding AdventurerGridTappedCommand}" />
-                </Grid.GestureRecognizers>
-                <Grid.RowDefinitions>
-                    <RowDefinition Height="0.7*" />
-                    <RowDefinition Height="0.3*" />
-                </Grid.RowDefinitions>
-                <Image x:Name="AdventurerImage" Grid.Row="0" Source="ic_adventurer" Aspect="AspectFit" Margin="10" />
-                <Label x:Name="AdventurerLabel" Grid.Row="1"
+        <views:NotificationExtensionView>
+            <views:NotificationExtensionView.ContentTemplate>
+                <Grid x:Name="MainGrid" RowSpacing="0" ColumnSpacing="0">
+                    <Grid x:Name="AdventurerGrid" BackgroundColor="Orange">
+                        <Grid.GestureRecognizers>
+                            <TapGestureRecognizer Command="{Binding AdventurerGridTappedCommand}" />
+                        </Grid.GestureRecognizers>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="0.7*" />
+                            <RowDefinition Height="0.3*" />
+                        </Grid.RowDefinitions>
+                        <Image x:Name="AdventurerImage" Grid.Row="0" Source="ic_adventurer" Aspect="AspectFit" Margin="10" />
+                        <Label x:Name="AdventurerLabel" Grid.Row="1"
                        Text="{helpers:Translate CharacterSelectionPage_Adventurer_Text}" TextColor="White"
                        FontSize="Medium" HorizontalTextAlignment="Center" VerticalTextAlignment="Center" />
-            </Grid>
-            <Grid x:Name="ProfessorGrid" BackgroundColor="Blue">
-                <Grid.GestureRecognizers>
-                    <TapGestureRecognizer Command="{Binding ProfessorGridTappedCommand}" />
-                </Grid.GestureRecognizers>
-                <Grid.RowDefinitions>
-                    <RowDefinition Height="0.7*" />
-                    <RowDefinition Height="0.3*" />
-                </Grid.RowDefinitions>
-                <Image x:Name="ProfessorImage" Grid.Row="0" Source="ic_professor" Aspect="AspectFit" Margin="10" />
-                <Label x:Name="ProfessorText" Grid.Row="1"
+                    </Grid>
```

Comment:
format

Reply:
Before the PR I went through all the files modified by me and applied our formatting so it should be all right. But I will check again and also remove the blank lines.